### ISSUE TITLE
No app suggestions in version based on 8.0

### ISSUE THREAD

**Issue by gcladores** [amirzaidi/Launcher3#112]

**Comment by AxiK101** [comment]
No app suggestions indeed..🤔

_Sent from my Sony F3211 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by GunnarAK** [comment]
This is a key feature I'm missing as well

**Comment by RikKoedoot** [comment]
Please read the FAQ before making these questions. In the FAQ amir says he doesn't use them because the code was to messy. If there is a good open source alternative he would think again

**Comment by amirzaidi** [comment]
It's in the test builds for 2.2 :)
I got a nice open source piece of code from another launcher

**Comment by GunnarAK** [comment]
So to get a testbuild running on my phone I have to download the source and build it myself, correct? Or are there ready-to-use apks?

**Comment by amirzaidi** [comment]
@GunnarAK there's a ready to use APK on the Telegram channel t.me/appforks

**Comment by GunnarAK** [comment]
Got it! Thanks
Never knew Telegram had these channels, but I don't use it at all
I have installed version uploaded sept 1st

**Comment by t1m0thyj** [comment]
I tried the test build from Sept 3rd that was on Telegram, which I thought should have the app suggestions feature, but after I installed it on my phone I didn't notice any difference from version 2.1.

I made sure that "App suggestions" was selected in my Google Now settings, so I'm not sure why they didn't show up with the test build. Do I need to wait 24 hours before the app suggestions start showing up or something like that?

**Comment by RikKoedoot** [comment]
@t1m0thyj this isn't google now launcher, so those settings don't change anything in this launcher, longpress on the homescreen, and then hit settings, then enable app suggestions

**Comment by t1m0thyj** [comment]
@rickkode Using the test build I tried long pressing on the homescreen and checking the settings there, but the only option there is to allow screen rotation. Do I need Oreo for the app suggestions option to show up?

**Comment by amirzaidi** [comment]
The app suggestions start coming up when you open apps through the app drawer only, it's not the advanced implementation that Google uses

**Comment by t1m0thyj** [comment]
I figured out why app suggestions weren't working for me. I didn't want to create a Telegram account, so I had asked someone if they could get the latest test build for me. It turned out I thought I was using a test build with the predictive apps commit added, but the APK file was identical to the one for version 2.1.

Today I created a Telegram account to download the latest test build. After installing it, the app suggestions started working right away.
